qms-tests/sanity/tests/documents/REQ-05.spec.ts

```diff
@@ -87,18 +90,23 @@ test.describe('ISO 13485, 4.2.4 Control of documents', () => {
     const category = faker.word.words(2)
 
     await leftSideMenuPage.clickButtonOnTheLeft('Documents')
-    await test.step('2. Create a new category and template', async () => {
+    await test.step('2. Create a new category', async () => {
       const documentContentPage = new DocumentContentPage(page)
       await documentContentPage.selectControlDocumentSubcategory('Categories')
       await documentContentPage.clickOnAddCategoryButton()
       await documentContentPage.fillCategoryForm(category, description, code)
-      await documentContentPage.clickNewDocumentArrow()
-      await documentContentPage.clickNewTemplate()
-      await createTemplateStep(page, title, description, category)
-      await documentContentPage.selectControlDocumentSubcategory('Templates')
-      await documentContentPage.chooseFilter(title, code)
+      await documentContentPage.checkIfCategoryIsCreated(category, code)
+      await test.step('3. Create a new template', async () => {
```

Comment:
Minor: should it be adjacent to `step(2)` instead of being nested?

Reply:
done :D